libs/server/Storage/Session/ObjectStore/SetOps.cs

```diff
@@ -429,5 +432,57 @@ public GarnetStatus SetMembers<TObjectContext>(byte[] key, ArgSlice input, ref G
         public GarnetStatus SetPop<TObjectContext>(byte[] key, ArgSlice input, ref GarnetObjectStoreOutput outputFooter, ref TObjectContext objectContext)
             where TObjectContext : ITsavoriteContext<byte[], IGarnetObject, SpanByte, GarnetObjectStoreOutput, long>
             => RMWObjectStoreOperationWithOutput(key, input, ref objectContext, ref outputFooter);
+
+        public GarnetStatus SetDiff<TObjectContext>(ArgSlice[] keys,out HashSet<byte[]> output, ref TObjectContext objectContext)
+            where TObjectContext : ITsavoriteContext<byte[], IGarnetObject, SpanByte, GarnetObjectStoreOutput, long>
+        {
+            output = default;
+            if (keys.Length == 0)
+                return GarnetStatus.OK;
+
+            output = [];
+            var comparer = new BytesComparer();
```

Comment:
I believe you can use [`ByteArrayComparer`](https://github.com/microsoft/garnet/blob/05643c36e636dd6914653e408aa27a9c73d3ddf4/libs/server/Resp/ByteArrayComparer.cs#L13) here.

Reply:
wow